finance/management/commands/xformnetfilerawdata.py

```diff
@@ -0,0 +1,365 @@
+"""
+Command to download and load California campaign finance data from Netfile.
+"""
+
+import datetime
+import time
+import warnings
+from dateutil.parser import parse as date_parse
+from numbers import Number
+
+import numpy as np
+import pandas as pd
+from django.db import transaction
+
+from ... import models
+from ballot.models import Ballot, BallotItemResponse
+from locality.models import City, State, ZipCode
+from netfile_raw.management.commands import downloadnetfilerawdata
+from office_election.models import Office, OfficeElection, Candidate
+from referendum.models import Referendum
+
+
+class Command(downloadnetfilerawdata.Command):
+    help = 'Download and load the netfile raw data into the clean database.'
+
+    def load(self):
+        """
+        According to:
+        https://github.com/caciviclab/caciviclab.github.io/wiki/Campaign-Finance
+
+        To get a comprehensive total of the amount of money raised for
+        any ballot measure, you must include:
+        * Form 460, Schedules A (monetary contributions) and
+            C (nonmonetary contributions), for all Primarily Formed and
+            Candidate Controlled committees
+        * Form 497, beginning from the ending period of the last Form 460 that
+            was filed, for all Primarily Formed and
+            Candidate Controlled committees
+        * Form 496, only expenses in support, only from committees that
+            are NOT the Primarily Formed or Candidate Controlled committees,
+            and only those beginning from the ending period of the
+            last Form 460 that was filed
+        """
+        if self.verbosity:
+            self.header("Loading disclosure data into database.")
+
+        self.data = pd.read_csv(self.combined_csv_path)
+        if self.verbosity:
+            print("There are %d rows in %s" % (
+                len(self.data), self.combined_csv_path))
+
+        # Check for any potentially missing data.
+        form_types = self.data['form_Type'].unique()
```

Comment:
It's native to the csv. I'd prefer to mirror the names in the CSV, rather than manipulate them. I worry that manipulating the column names to be lowercase could confuse people familiar with the data. And it'd be a pain in the butt :)

If you feel this is an important point, I can convert the column labels to lowercase, just lemme know.